doc/version-history.rst

```diff
@@ -14,6 +14,7 @@ v1.2.0
 * Renamed to lowercase linearstage namespace.
 * Make everything relative imports.
 * Update config_schema to use enums to divide device configurations.
+* Use generate_pre_commit_conf and DevelopPipeline.
 
 v1.1.0

```

Comment:
There's currently only `v1.0.0` and `v1.0.1` release tags. Did you forgot to release `v1.1.0`? this is supposed to be `v1.2.0`. Make sure you release the code!